### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by dzambrano**

### CODE REVIEW

File: bowlrelease/utils/utils.py [hunk 1]
```diff
@@ -61,3 +64,98 @@ def extract_features(dataloader, device, filename="features.npy"):
...
+    preds = np.concatenate(pred_)
+    np.save(filename, preds)
+
+
+def extract_all_videos_features(
+    data_dir: str,
+    annotations_dir: str,
+    video_dir: str,
+    device: str,
+    override: bool = True,
```
Comment (jaydeepde) [inline bowlrelease/utils/utils.py]: Not clear what this override is doing. For this function write Args and Return

File: bowlrelease/runner/trainer.py [hunk 2]
```diff
@@ -24,73 +26,128 @@ def train(dataloader, model, loss_fn, optimizer, device):
...
-        optimizer = torch.optim.AdamW(
-            [
-                {"params": model.rn_model.parameters(), "lr": 1e-5},
-                {"params": model.feat_bn.parameters()},
-                {"params": model.classifier.parameters()},
-            ],
-            lr=1e-3,
-        )
+
+    loss_fn = torch.nn.MSELoss()
```
Comment (jaydeepde) [inline bowlrelease/runner/trainer.py]: Why MSE Loss if you are doing binary classification

**Review by jaydeepde — APPROVED** [review]
Looks good to me!

File: bowlrelease/utils/utils.py (hunk 1 above)
Comment (dzambrano) [reply bowlrelease/utils/utils.py]: added doc

File: bowlrelease/runner/trainer.py (hunk 2 above)
Comment (dzambrano) [reply bowlrelease/runner/trainer.py]: moved to bce
